site/app/entities/UserEntity.php

```diff
@@ -133,8 +133,8 @@ public function accessGrading(): bool {
      */
     public function getDisplayInfo(): array {
         $out = [];
-        $out["given_name"] = (strlen($this->user_preferred_givenname ?? "") > 0) ? $this->user_preferred_givenname : $this->user_givenname;
-        $out["family_name"] = (strlen($this->user_preferred_familyname ?? "") > 0) ? $this->user_preferred_familyname : $this->user_familyname;
+        $out["given_name"] = is_null($this->user_preferred_givenname) ? $this->user_preferred_givenname : $this->user_givenname;
+        $out["family_name"] = is_null($this->user_preferred_familyname) ? $this->user_preferred_familyname : $this->user_familyname;
```

Comment:
Can use null coalescing operator.
```suggestion
        $out["family_name"] = $this->user_preferred_familyname ?? $this->user_familyname;
```

Reply:
Resolved in [f6d4fef](https://github.com/Submitty/Submitty/pull/11376/commits/f6d4fef4754704da4c1bbe065bbb23fd29da0688)